### ISSUE TITLE
vue3 support

### ISSUE THREAD

**Issue by toinbis** [XiongAmao/vue-easy-lightbox#51]
Vue3 is out. Any plans to support it?

Thanks

**Comment by XiongAmao** [comment]
I'm learning Vue 3.0. 
And, Yes, it will support Vue 3.0 later!

**Comment by XiongAmao** [comment]
Hi , VueEasyLitghtbox based on Vue.js 3.0 has been released (v1.0.0-rc.1), you can try it out. 
Note that Vue.js 2.0 is not supported.

**Comment by XiongAmao** [comment]
`vue-easy-lightbox@0.x` supports `Vue.js 2`  , documents [here](https://github.com/XiongAmao/vue-easy-lightbox/tree/vue2.x) . 

```shell
npm install vue-easy-lightbox@vue2
# or 
yarn add vue-easy-lightbox@vue2
```
